language/move-model/src/pragmas.rs

```diff
@@ -161,6 +165,7 @@ pub fn is_property_valid_for_condition(kind: &ConditionKind, prop: &str) -> bool
             | CONDITION_EXPORT_PROP
             | CONDITION_ABSTRACT_PROP
             | CONDITION_CONCRETE_PROP
+            | CONDITION_SUSPENDABLE_PROP
```

Comment:
Nitpicking: I think it is probably better to move this to line 177 below.

Reply:
Why's that?  I will leave for now.

Reply:
These are properties valid for something like `ensures [prop] P` and not for this one you are introducing.

Reply:
NM this is a property of this kind.

Reply:
Leaving it here will allow us to write sth like
`ensures [suspendable] P;` or `aborts_if [suspendable] P;`,
which does not hurt but also can be confusing.

Moving it later will make the `move-model` throw an error if it sees an `ensures [suspendable] P;`